instance/models/openedx_appserver.py

```diff
@@ -176,6 +176,10 @@ class Meta:
             "behind a firewall. (In the django admin, separate group names with a comma.)"
         )
     )
+    enable_vm_dns_records = models.BooleanField(
+        default=False,
+        help_text="Set additional DNS records pointing directly to the active app servers."
+    )
```

Comment:
@smarnach OK - can you put a follow-up story to remove the option and make this the standard behavior? Also worth checking with me in the future before putting features behind configuration options or flag - having this on all instances was part of the expected outcomes of that task, and one of the reasons the support request from Harvard was addressed this way.

Thanks!